pyproject.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0.0"
+pydantic = "1.10.12"
```

Comment:
Let's not do this. OAK is a core library. People are importing OAK in combination with other libraries. There is nothing in OAK that depends on 1.10.12 specifically. However, there is a need to pin < 2 due to transitive dependencies not being pinned correctly

Reply:
Fixed